### ISSUE TITLE
About the test result

### ISSUE THREAD

**Issue by whyandbecause** [root0yang/BlindNet#1]
Thank you for your intersting work. I retrained Blindnet, but the results are much worse than the paper.  I used python=3.8, pytorch=2.2.1, cuda=12.1. Also, there were some bugs when running the code, which I fixed:
change '''import scipy.misc as m''' to '''import imageio as m'''. What might be the problem? The results are as follows:
> Dataset name: gtav
>
>best record: [dataset name gtav], [val loss 0.16470], [acc 0.94620], [acc_cls 0.79138], [mean_iu 0.70810], [fwavacc 0.90182], [epoch 26]
> Dataset name: cityscapes
>
> [epoch 26], [dataset name cityscapes], [val loss 0.54270], [acc 0.83315], [acc_cls 0.51697], [mean_iu 0.41233], [fwavacc 0.73179]
>
> Dataset name: bdd100k
>
> [epoch 26], [dataset name bdd100k], [val loss 0.58890], [acc 0.83065], [acc_cls 0.50769], [mean_iu 0.38534], [fwavacc 0.73354]
>
> Dataset name: mapillary
>
> [epoch 26], [dataset name mapillary], [val loss 0.43129], [acc 0.86271], [acc_cls 0.59402], [mean_iu 0.46410], [fwavacc 0.78339]<br>

**Comment by root0yang** [comment]
Thank you for your interest in our work.

It seems that the difference in performance might be related to the version of PyTorch and other dependencies used during your retraining process. Code we provide is optimized for PyTorch version 1.2. Using different versions of PyTorch can result in varying performance outcomes.

Additionally, the issue with scipy.misc that you encountered is indeed related to the version of SciPy.
